data/schema/core/conditionalwml.cfg

```diff
@@ -25,17 +25,17 @@
 		max=infinite
 		{INSERT_TAG}
 		{SIMPLE_KEY name string}
-		{SIMPLE_KEY contains string}
-		{SIMPLE_KEY equals string}
-		{SIMPLE_KEY not_equals string}
-		{SIMPLE_KEY numerical_equals string}
-		{SIMPLE_KEY numerical_not_equals string}
-		{SIMPLE_KEY greater_than string}
-		{SIMPLE_KEY greater_than_equal_to string}
-		{SIMPLE_KEY less_than string}
-		{SIMPLE_KEY less_than_equal_to string}
-		{SIMPLE_KEY boolean_equals string}
-		{SIMPLE_KEY boolean_not_equals string}
+		{SIMPLE_KEY contains any}
+		{SIMPLE_KEY equals any}
+		{SIMPLE_KEY not_equals any}
+		{SIMPLE_KEY numerical_equals s_int}
+		{SIMPLE_KEY numerical_not_equals s_int}
+		{SIMPLE_KEY greater_than s_int}
+		{SIMPLE_KEY greater_than_equal_to s_int}
+		{SIMPLE_KEY less_than s_int}
+		{SIMPLE_KEY less_than_equal_to s_int}
+		{SIMPLE_KEY boolean_equals s_bool}
+		{SIMPLE_KEY boolean_not_equals s_bool}
```

Comment:
Changing these to s_int and s_bool should maybe be a separate commit? It has nothing to do with the tstring stuff.